src/lib/Scenes/Fair/Components/FairHeader/index.tsx

```diff
@@ -76,7 +125,16 @@ export class FairHeader extends React.Component<Props> {
           </CountdownContainer>
         </BackgroundImage>
         <Spacer m={1} />
-
+        <Box mx={2}>
+          {this.getContextualDetails(
+            artistCount,
+            firstArtist,
+            lastArtist,
+            exhibitorCount,
+            firstExhibitor,
+            lastExhibitor
+          )}
```

Comment:
Rather than extracting data from props and passing it into a method that returns something right back, best to just pull the data from props from within the method itself. So the render method becomes super clean: 

```jsx
<Box mx={2}>
  {this.getContextualDetails()}
</Box>
```
And then in the method itself, grab the data since it can be easily accessed via `this.props`. This reduces complexity: 
```js
getContextualData() {
  const { keep all data colocated where its needed } = this.props
  ...
}